src/node/nodetonode.h

```diff
@@ -157,8 +160,7 @@ namespace ccf
       std::vector<uint8_t> cipher(data.size());
       n2n_channel.encrypt(hdr, asCb(msg_hdr), data, cipher);
```

Comment:
Why do we need to both encrypt and then integrity protect the entire msg? this will produce considerable overhead that is not required